extensions/wikia/Optimizely/Optimizely.class.php

```diff
@@ -6,6 +6,14 @@
  *
  */
 class Optimizely {
+
+	/**
+	 * Which environments are considered a developemnt/testing environments for Optimizely experiments?
+	 * Optimizely Wikia Dev project script should be served also on sandboxes, to prevent poluting Wikia Prod project
+	 * script with experiments that are still under developement.
+	 */
+	const OPTIMIZELY_DEV_ENVIRONMENTS = [ WIKIA_ENV_DEV, WIKIA_ENV_SANDBOX ];
```

Comment:
Sandboxes are using production accounts, DBs, etc. When we introduce exception like this we make Sandbox environment inconsistent.
Finally mercury uses production Optimizely account on sandbox which makes things even more complicated. https://github.com/Wikia/mercury/blob/dev/config/localSettings.base.ts#L50